src/app/components/HoldPage/HoldConfirmation.jsx

```diff
@@ -93,18 +93,23 @@ class HoldConfirmation extends React.Component {
               </p>
               <p>Regular hours:</p>
               <table className="generic-table">
+                <caption className="visuallyHidden">Location hours</caption>
                 <tbody>
+                  <tr>
+                    <th scope="col">Day</th>
```

Comment:
Tables need headings for columns. Visually hiding them messes up the top border so I left it for now.